doc/README.md

```diff
@@ -0,0 +1,103 @@
+The Coq documentation
+=====================
+
+The Coq documentation includes
+
+- A Reference Manual
+- A document presenting the Coq standard library
+
+The documentation of the latest released version is available on the Coq
+web site at [coq.inria.fr/documentation](http://coq.inria.fr/documentation).
+
+Additionnally, you can view the documentation for the current master version at
+<https://gitlab.com/coq/coq/-/jobs/artifacts/master/file/_install_ci/share/doc/coq/sphinx/html/index.html?job=documentation>.
+
+The reference manual is written is reStructuredText and compiled
+using Sphinx (see [`sphinx/README.rst`](/doc/sphinx/README.rst))
+to learn more about the format that is used.
+
+The documentation for the standard library is generated from
+the `.v` source files using coqdoc.
+
+Prerequisite
+------------
+
+To produce the complete documentation in HTML, the following tools are required:
+
+  - Python 3
+  - Sphinx 1.6.5 (http://www.sphinx-doc.org/en/stable/)
+  - sphinx_rtd_theme
+  - pexpect
+  - beautifulsoup4
+  - Antlr4 runtime for Python 3
+
+To produce the documentation in PDF and PostScript formats the following
+additional tools are required:
+
+  - latex (latex2e)
+  - pdflatex
+  - dvips
+  - makeindex
+
+Under recent Debian based operating systems (Debian 10 "Buster",
+Ubuntu 18.04, ...) a working set of packages for compiling the
+documentation for Coq is:
+
+    texlive-latex-extra texlive-fonts-recommended
+    python3-sphinx python3-pexpect python3-sphinx-rtd-theme
+    python3-bs4 python3-sphinxcontrib.bibtex python3-pip
```

Comment:
Would it be correct/useful to refer to these as "python packages"?  Also installed with pip?  Or are they some kind of Debian-specific packages?

Reply:
All the ones starting with `python3` are Python packages and can be installed using pip.